jmx-metrics/src/test/java/io/opentelemetry/contrib/jmxmetrics/JmxConfigTest.java

```diff
@@ -16,7 +16,8 @@
   @Test
   void staticValues() {
     assertThat(JmxConfig.AVAILABLE_TARGET_SYSTEMS)
-        .containsOnly("cassandra", "jvm", "kafka", "kafka-consumer", "kafka-producer", "tomcat");
+        .containsOnly(
+            "cassandra", "jvm", "kafka", "kafka-consumer", "kafka-producer", "tomcat", "activemq");
```

Comment:
nit, alphabetical order is desirable here and below

Reply:
gotcha, will implement